Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -148,6 +151,33 @@ public static ProcessService getInstance() {
         return convertJSONObjectsToDTOs(findAllDocuments(sort, offset, size), false);
     }
 
+    @Override
+    public List<ProcessDTO> findAll(String sort, Integer offset, Integer size, Map filters) throws DataException {
+
+        if (Objects.equals(filters, null) || filters.isEmpty()) {
+            return findAll(sort, offset, size);
```

Comment:
Even if filters are empty, it should still distinguish between processes and templates.